packages/smooth_app/lib/generic_lib/buttons/smooth_large_button_with_icon.dart

```diff
@@ -58,6 +58,8 @@ class SmoothLargeButtonWithIcon extends StatelessWidget {
             child: AutoSizeText(
               text,
               maxLines: 2,
+              minFontSize: 10,
+              stepGranularity: 2,
```

Comment:
What's wrong with the default `stepGranularity` of 1?

Reply:
Will change it to 1.

Reply:
> Will change it to 1.

In that case please leave the default value and do not set it.

Reply:
Ok